CHANGES.rdoc

```diff
@@ -4,6 +4,10 @@ This document describes the relevant changes between releases of the
 _hawkular-client_ project.
 
 
+=== v 3.0.3
+* Standardized Exceptions under the Hawkular namespace, and the old names were deprecated, here is the list:
+1. <code>Hawkular::BaseClient::HawkularException</code> -> <code>Hawkular::Exception</code>
+2. <code>Hawkular::BaseClient::HawkularConnectionException</code> -> <code>Hawkular::ConnectionException</code>
```

Comment:
Its a thin line. 
Before we were raising RuntimeError exceptions and now we are raising Hawkular::ArgumentError.

Still, If a tree falls in a forest and no one is around to hear it, does it make a sound?

[1] https://github.com/hawkular/hawkular-client-ruby/pull/222/files#diff-27ed7a1113ec9bf87df507d3990b581eR40